recipes-support/cryptominisat/cryptominisat_git.bb

```diff
@@ -0,0 +1,26 @@
+DESCRIPTION = "CryptoMiniSat is an advanced incremental SAT solver."
+HOMEPAGE = " https://www.msoos.org"
+LICENSE = "MIT"
+SECTION = "devel/verilog"
+
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=7cdd497f1ddb031d004fad321c45d8df"
+SRC_URI = "git://github.com/msoos/cryptominisat;protocol=https"
+SRCREV = "e440d535cb4500706b38ec5d655111a735dbb60e"
+
+S = "${WORKDIR}/git"
+
+PV = "5.6.8+git${SRCPV}"
+
+inherit cmake python3native
+
+EXTRA_OECMAKE = "-DENABLE_PYTHON_INTERFACE=OFF -DSTATICCOMPILE=ON -DNOM4RI=ON -DONLY_SIMPLE=ON"
+
+SRC_URI += " \
+    file://0001-CMakeLists.txt-Remove-check-for-float-precision.patch \
```

Comment:
It looks like this patch is not intended to be upstreamed?

Also it should be possible to generate the cache variable value and insert it into `toolchain.cmake` such that the patch is not required. Something like:

```
cmake_do_generate_toolchain_file_append() {
    cat >> ${WORKDIR}/toolchain.cmake <<EOF
set(HAVE__FPU_SETCW FALSE CACHE FORCE)
EOF
}
```

It could also be conditionally populated as true for i386/x86 via a recipe variable, if having it provides some sort of performance benefit.

Reply:
The patch is cross-compile specific. It removes the check_float_precision() call because it tries to run some binary on the build host to determine fpu capabilities. I didn't find a way yet to implement the check at build time, instead of running the binary on the host. So I don't have anything worth upstreaming.

I am therefore inclined to drop the patch and use the "cmake_do_generate_toolchain_file_append" approach instead.